doc/man/8/ceph-bluestore-tool.rst

```diff
@@ -106,7 +107,10 @@ Commands
 
 :command:`show-label` --dev *device* [...]
 
-   Show device label(s).	   
+   Show device label(s). Label is a special 4K area that plays the role of superblock,
+   provides metadata that exposes version, creation time, size, etc.
+   There are multiple copies of the block at offsets: 0, 1G, 10G, 100G and 1000G.
+   As an exception, label can be printed even if an OSD is running.
```

Comment:
The label is a 4KB structure at the beginning of the device that serves as a superblock, holding metadata including version, creation time, size, etc.  There are copies at offsets 1GB, 10GB, 100GB, and 1000GB.  The label may be printed while an OSD is running.

Reply:
Isn't this redundant wrt existing text around lines 203 / 211?

Reply:
it feels to me this change should actually be moved to section `Device labels`

Reply:
Right.
I will move detailed explanation around "Device labels" section.

Reply:
@anthonyeleven Please recheck. My PRs always benefit from your language input.